bin/_mocha

```diff
@@ -199,7 +207,8 @@ program
   .option('--delay', 'wait for async suite definition')
   .option('--allow-uncaught', 'enable uncaught errors to propagate')
   .option('--forbid-only', 'causes test marked with only to fail the suite')
-  .option('--forbid-pending', 'causes pending tests and test marked with skip to fail the suite');
+  .option('--forbid-pending', 'causes pending tests and test marked with skip to fail the suite')
+  .option('--file [file]', 'include a file to be ran during the suite', collect, []);
```

Comment:
the parameter to `--file` is required, so I think this needs to be `--file <file>`.